### ISSUE TITLE
Eclipse bnd bundle path seems to ignore bnd files outside of eclipse workspace

### ISSUE THREAD

**Issue by jwausle** [bndtools/bndtools#1771]
Eclipse bnd bundle path seems to ignore bnd files outside of eclipse workspace.
- the bnd cmdline works correct.

*Howto reproduce*:
```
git clone https://github.com/jwausle/bndtools-testcase-fragment.git --branch eclipse-bundle-path-outdated
cd bndtools-testcase-fragment
sh gradlew clean build
   ... (build output) ... 
BUILD SUCCESSFUL in 45s
```

open eclipse in new workspace 
- import projects (successful build expected)
- edit build.bnd  (not cnf/build.bnd)
  change:'p2.version=1.1.2'
  to    :'p2.version=1.2.0'
- still successful build (but the build should fail)
- like at cmdline

```
sh gradlew clean build
   ... (build output) ...
BUILD FAILED   
```

*Reason for failed build*
- the change should force higher version of `-buildpath: com.qivicon.common.messaging;version='[1.1,1.2)'`
  in: depends.on.p2/bnd.bnd

*Environment*
- eclipse oxygen
- bndtools 3.4
- macos sierra

*Quickfix*:

```
rm -rf cnd/cache/p2-*
reimport the project again into eclipse workspace
```

**Comment by jwausle** [comment]
Maybe relation to https://github.com/bndtools/bndtools/issues/1770

**Comment by jwausle** [comment]
Fixed with bnd-3.5.
